opensrp-opd/src/main/java/org/smartregister/opd/adapter/ClientLookUpListAdapter.java

```diff
@@ -38,8 +38,9 @@ public void onBindViewHolder(@NonNull MyViewHolder holder, int i) {
         String firstName = Utils.getValue(commonPersonObject.getColumnmaps(), "first_name", true);
         String lastName = Utils.getValue(commonPersonObject.getColumnmaps(), "last_name", true);
         String nationalId = Utils.getValue(commonPersonObject.getColumnmaps(), "national_id", true);
+        String opensrpId = Utils.getValue(commonPersonObject.getColumnmaps(), "opensrp_id", true);
```

Comment:
Can you move this strings to single reference point. I am guessing they are constants or can be refactored to be configurable?